### DISCUSSION TITLE
JS in Wiki

### DISCUSSION THREAD

**Discussion by WikiJS-pixel** [requarks/wiki#7364] [Help / Questions] [answered]
Hello everyone!

I really don't understand how to enable scripts in Wiki. I turned off Sanitize HTML in Rendering > html html > Security, I finally got scripts to work inserting them in the Theme section of the Administration. 

The main question is how to get the scripts to work in the script tab of the page? I've tried every method I could find, including [this](https://docs.requarks.io/troubleshooting#how-to-execute-javascript-code-on-page-load).

**Comment by NGPixel** [comment]
Do you have the "Write scripts" permissions enabled under your group permissions?

**Reply by WikiJS-pixel** [reply]
Yes, I'm admin.

**Reply by WikiJS-pixel** [reply, marked as answer]
It's my fault, the js in the theme interfered with js in the script tap of the page, spamming errors. Really sorry to bother you.
